content/influxdb/v2.0/reference/cli/influx/_index.md

```diff
@@ -67,18 +47,70 @@ influx [command]
 | [template](/influxdb/v2.0/reference/cli/influx/template)     | Summarize and validate an InfluxDB template          |
 | [transpile](/influxdb/v2.0/reference/cli/influx/transpile)   | Manually transpile an InfluxQL query to Flux         |
 | [user](/influxdb/v2.0/reference/cli/influx/user)             | User management commands                             |
+| [v1](/influxdb/v2.0/reference/cli/influx/v1)                 | Work with the v1 compatibility API                   |
 | [version](/influxdb/v2.0/reference/cli/influx/version)       | Print the influx CLI version                         |
 | [write](/influxdb/v2.0/reference/cli/influx/write)           | Write points to InfluxDB                             |
 
-## Mapped environment variables
+## Flags
+
+| Flag |          | Description                   |
+|:---- |:---      |:-----------                   |
+| `-h` | `--help` | Help for the `influx` command |
+
+## Patterns and conventions
+The `influx` CLI utilizes the following patterns and conventions:
+
+- [Easily provide required authentication credentials](#easily-provide-required-authentication-credentials)
```

Comment:
I agree it's a pattern. My main thought was if someone lands on one of these CLI docs, do they know they need to do something if they haven't already otherwise the CLI won't work? A section called "patterns and conventions" may be interpreted as "standards and guidelines" reference material rather than something you must take action on.